test/components/component_test.rb

```diff
@@ -111,7 +111,11 @@ def test_all_components_support_inline_styles
     default_args = { style: "width: 100%;" }
     COMPONENTS_WITH_ARGS.each do |component, args, proc|
       render_component(component, default_args.merge(args), proc)
-      assert_selector("[style='width: 100%;']", visible: :all)
+      if component == Primer::Alpha::Tooltip
+        assert_selector("[style='width: 100%;visibility: hidden']", visible: :all)
+      else
+        assert_selector("[style='width: 100%;']", visible: :all)
+      end
```

Comment:
It looks like `#assert_matches_style` is designed to be called on a Capybara `Node` object. However it also looks like `assert_selector` supports the `style:` argument, so maybe this works?

```suggestion
      assert_selector(style: { "width" => "100%" })
```

Reply:
Hmm this syntax appears correct but looks like something could be broken:

```
Nokogiri::CSS::SyntaxError: unexpected '$' after ''
    (eval):3:in `_racc_do_parse_c'
    (eval):3:in `do_parse'
```

Reply:
I think we might need to do somethign gross like:

```.rb
rendered = render_component(component, default_args.merge(args), proc)
assert_equal(true, rendered.inner_html.include?('style="width: 100%;'))
```

Reply:
Hmm that's a good idea. You should be able to use `page.body` I think? The result of the render is also available as `@rendered_content` or `rendered_content`, but that's technically part of the private API.